bin/run-demo.sh

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/env bash
-
-while getopts ":v:n:i:" opt
-do
-  case ${opt} in
-    v) VOLUME_SOURCE=${OPTARG};;
-    n) NAME=${OPTARG};;
-    i) DOCKER_IMAGE=${OPTARG};;
-    h)
-        echo "Usage: run-demo [-h] -v <VOLUME_SOURCE> -n <NAME> -i <DOCKER_IMAGE>"
-        exit 0
-        ;;
-    \?)
-        echo "Invalid option: -$OPTARG" 1>&2
-        exit 1
-        ;;
-    : )
-        echo "Invalid option: -$OPTARG requires an argument" 1>&2
-        exit 1
-        ;;
-  esac
-done
-
-if ((OPTIND == 1))
-then
-    echo "No options specified"
-    exit 1
-fi
-
-shift $((OPTIND - 1))
-
-docker run \
-    --name ${NAME} \
-    --volume ${VOLUME_SOURCE}:/project \
-    --env COLUMNS=110 \
-    --env LINES=100 \
-    ${DOCKER_IMAGE} \
-    process /project/rector_analyzed_file.php \
-    --autoload-file /project/rector_analyzed_file.php \
```

Comment:
This can now removed, right?

Reply:
Yep, deleting this was not an accident :-)

Reply:
That would be nice accident :D I meant the `--autoload-file` option :)

With static reflection, the autoload file does not seems useful anymore

Reply:
I see, anyways it is no longer used anywhere so all good 👌